stata_kernel/stata_lexer.py

```diff
@@ -46,17 +44,33 @@ class StataLexer(RegexLexer):
         'mata': [
             include('strings'),
             (r'^[^\n]*?\{', Token.MatchingBracket.Other, 'block'),
-            (r'^[^\n]*?\(', Token.MatchingBracket.Other, 'paren'),
-            (r'[\r\n][^\r\n\S]*end(?=(\s|[^\s\w\.]).*?$|$)', Token.Aborted, '#pop'),
+            (r'^[^\r\n]*?\(', Token.MatchingBracket.Paren, 'paren'),
+            (r'[\r\n][^\r\n\S]*end(?=(\s|[^\s\w\.]).*?$|$)', Token.Mata.Close, '#pop'),
```

Comment:
How do you get arbitrary names?? I wouldn't be using `Token.MatchingBracket.Other` if I knew how to create my own Tokens!

Reply:
I didn't know you could use arbitrary names until earlier today. I kept getting errors so I looked it up. It turns out you can just name them `Token.WhateverTheHellYouWant` and it will take it.

Reply:
Amazing. I may submit a PR to change the names, because `Token.MatchingBracket.Other` should be named like `Token.Block`

Reply:
I think at one point I use `startswith.('Token.MatchingBracketOther')`. I can just change it to `Token.Block.Paren` and it's all the same.